recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,116 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import get
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd, cross_building, stdcpp_library
+
+class MysqlConnectorCPPRecipe(ConanFile):
+    name = "mysql-connector-cpp"
+    license = "GPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://dev.mysql.com/doc/connector-cpp/8.0"
+    description = "A Conan package for MySQL Connector/C++ with OpenSSL, Boost, and libmysqlclient"
+    topics = ("mysql", "connector", "cpp", "openssl", "boost", "libmysqlclient", "jdbc", "static")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _min_cppstd(self):
+        return "17"
```

Comment:
Just to be sure, is c++17 mandatory for building mysql-connector-cpp ?

Reply:
Yes, in the main CMakeLists.txt you will find:

```

#
# Compiler settings
# =================
#

enable_pic()
enable_cxx17()
```

That function passes `-std=c++17` to cxx flags.